src/components/Accordion/Accordion.tsx

```diff
@@ -1,22 +1,26 @@
 import * as React from "react";
-import Button, { ButtonOptions } from "../Button/Button";
+import Button from "../Button/Button";
 import bem from "../../utils/bem";
 import { ButtonTypes, ButtonIconPositions } from "../Button/ButtonTypes";
 
 export interface AccordionProps {
+    /** ID that other components can cross reference for accessibility purposes */
     id?: string;
 
+    /** String labelling the button that opens the accordion */
+    accordionLabel?: string;
```

Comment:
we're sure about this, right?  We explicitly want to exclude <em> or <strong> or spans or icons?

Reply:
Good call... I actually think it should accept a variety of things since I think SCC might use an Accordion for the Subject Header Explorer (where the accordion innards look like columns to me)